common/commands/config_test.go

```diff
@@ -49,7 +49,14 @@ func TestArtifactorySshKey(t *testing.T) {
 
 func TestAccessToken(t *testing.T) {
 	inputDetails := tests.CreateTestServerDetails()
-	inputDetails.AccessToken = "accessToken"
```

Comment:
Notice that you deleted the access token's content. Is it intended?
Same question for the new TestAccessTokenWithUsername test.